src/handlers.cpp

```diff
@@ -343,14 +343,6 @@ void ImpiTask::on_mar_response(Diameter::Message& rsp)
   uint32_t vendor_id = 0;
   _maa->experimental_result(experimental_result_code, vendor_id);
 
-  if (result_code != 0)
-  {
-    mar_results_tbl->increment(SNMP::DiameterAppId::BASE, result_code);
-  }
-  else if (experimental_result_code != 0 && vendor_id == VENDOR_ID_3GPP)
-  {
-    mar_results_tbl->increment(SNMP::DiameterAppId::_3GPP, experimental_result_code);
-  }
```

Comment:
I understand from @bossmc that IMS mandates that responses have exactly one of these, so the code here is actually fine. It's probably worth re-instating it given it's a fair bit simpler than the other option.